src/Codeception/Lib/Driver/Db.php

```diff
@@ -247,91 +253,86 @@ protected function generateWhereClause(array &$criteria)
     }
 
-    public function getQuotedName($name)
+    public function getQuotedName($name): string
     {
         return '"' . str_replace('.', '"."', $name) . '"';
     }
 
-    protected function sqlLine($sql)
+    public function sqlLine($sql): bool
     {
         $sql = trim($sql);
         return (
             $sql === ''
             || $sql === ';'
-            || preg_match('~^((--.*?)|(#))~s', $sql)
+            || preg_match('#^((--.*?)|(\#))#s', $sql)
         );
     }
 
-    protected function sqlQuery($query)
+    public function sqlQuery($query): void
     {
         try {
             $this->dbh->exec($query);
-        } catch (\PDOException $e) {
+        } catch (PDOException $pdoException) {
             throw new ModuleException(
-                'Codeception\Module\Db',
-                $e->getMessage() . "\nSQL query being executed: " . $query
+                \Codeception\Module\Db::class,
+                $pdoException->getMessage() . "\nSQL query being executed: " . $query
             );
         }
     }
 
-    public function executeQuery($query, array $params)
+    public function executeQuery($query, array $params): PDOStatement
     {
-        $sth = $this->dbh->prepare($query);
-        if (!$sth) {
-            throw new \Exception("Query '$query' can't be prepared.");
+        $pdoStatement = $this->dbh->prepare($query);
+        if (!$pdoStatement) {
+            throw new Exception("Query '{$query}' can't be prepared.");
         }
 
         $i = 0;
-        foreach ($params as $value) {
-            $i++;
-            if (is_bool($value)) {
-                $type = \PDO::PARAM_BOOL;
-            } elseif (is_int($value)) {
-                $type = \PDO::PARAM_INT;
+        foreach ($params as $param) {
+            ++$i;
+            if (is_bool($param)) {
+                $type = PDO::PARAM_BOOL;
+            } elseif (is_int($param)) {
+                $type = PDO::PARAM_INT;
             } else {
-                $type = \PDO::PARAM_STR;
+                $type = PDO::PARAM_STR;
             }
-            $sth->bindValue($i, $value, $type);
+            $pdoStatement->bindValue($i, $param, $type);
         }
 
-        $sth->execute();
-        return $sth;
+        $pdoStatement->execute();
+        return $pdoStatement;
     }
 
     /**
-     * @param string $tableName
-     *
-     * @return array[string]
+     * @return string[]
      */
-    public function getPrimaryKey($tableName)
+    public function getPrimaryKey(string $tableName): array
     {
         return [];
     }
 
-    /**
-     * @return bool
-     */
-    protected function flushPrimaryColumnCache()
+    protected function flushPrimaryColumnCache(): bool
     {
         $this->primaryKeys = [];
 
         return empty($this->primaryKeys);
     }
 
-    public function update($table, array $data, array $criteria)
+    public function update($table, array $data, array $criteria): string
```

Comment:
all $table params are strings.

Some methods take $tableName, other take $table, maybe it would be better to pick one name.